docs/docs/sourcing-from-agilitycms.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Sourcing from Agility CMS
+---
+
+This guide takes you through the steps involved in setting up a Gatsby site that fetches content from [Agility CMS](https://agilitycms.com/).
+
+## What is Agility CMS? What makes it different?
+
+[Agility CMS](https://agilitycms.com/) is a headless Content Management System (CMS) that let's you easily define your custom content types and relationships easily.  This is called Content Architecture, and you can reuse this content for your websites and apps.
+
+In addition, Agility CMS provides a page routing API, which allows you to offload control of the sitemap to the content editors.
+
+All content is available through the Agility CMS Fetch or Preview API.
+
+
+## Getting Started
+
+### Create A Free Agility Account
+It only takes a minute to create an Agility CMS account, and it's free forever. [Sign up here](https://account.agilitycms.com/sign-up?product=agility-free).  
```

Comment:
Avoid time so no one is frustrated if he takes longer times. 
Make more clear there many plans and not always free forever. 

```suggestion
Create an Agility CMS account with the Free Plan (this plan is free forever). [Sign up to Agility CMS](https://account.agilitycms.com/sign-up?product=agility-free).  
```